tests/e2e/specs/subscriptions/README.md

```diff
@@ -0,0 +1,22 @@
+# WooCommerce Payments e2e tests - WooCommerce Subscriptions
```

Comment:
What do you think of mentioning this on the [e2e README Setup](https://github.com/Automattic/woocommerce-payments/tree/develop/tests/e2e#setup)?

It might be confusing if someone following those instructions hits an error due to not configuring the Subscriptions-specific variables.

I think something simple should be enough, like:

```
If you have access to the Subscriptions plugin, please follow the instructions from tests/e2e/specs/subscriptions/README.md before running the setup.
If you don't, you may skip Subscriptions setup and tests by adding `SKIP_WC_SUBSCRIPTIONS_TESTS=1` to your `local.env' in the `tests/e2e/config` folder.
```

Reply:
That's a great idea. I added these under the first step in `For local setup:`, but let me know if you feel there might be a better place to put it.

Reply:
I think the place you picked is great! There's just some small issue with the indentation, which leads to the list breaking up a bit. Adding four spaces before the `*` should be enough to fix it.

![image](https://user-images.githubusercontent.com/7714042/118039692-fede5b80-b346-11eb-91f9-72231dc171b2.png)

Reply:
Woops, nice catch, pushed up that fix.

Reply:
Fantastic! Thank you for the quick fixes, I'll approve this once the checks are successful 😄

Reply:
Oh no worries at all, I appreciate the helpful feedback!